src/components/Users/UserAvatar.tsx

```diff
@@ -99,21 +101,32 @@ export default function UserAvatar({
               className="h-20 w-20"
             />
             <div className="my-4 ml-4 flex flex-col gap-2">
-              <ButtonV2
-                onClick={(_) => setEditAvatar(!editAvatar)}
-                type="button"
-                id="change-avatar"
-                className="border border-gray-200 bg-gray-50 text-black hover:bg-gray-100"
-                shadow={false}
-                disabled={!showAvatarEdit(authUser, userData)}
-                tooltip={
-                  !showAvatarEdit(authUser, userData)
-                    ? t("edit_avatar_permission_error")
-                    : undefined
-                }
-              >
-                {t("change_avatar")}
-              </ButtonV2>
+              {!showAvatarEdit(authUser, userData) ? (
+                <TooltipComponent
+                  content={t("edit_avatar_permission_error")}
+                  className="w-full"
+                >
+                  <Button
+                    variant="white"
+                    onClick={(_) => setEditAvatar(!editAvatar)}
```

Comment:
You could ignore `_` if you ain't gonna use it anyways.

```suggestion
                    onClick={() => setEditAvatar(!editAvatar)}
```